### PR TITLE
Fixed dockerfile build failed caused by missing libcuda.so

### LINKED ISSUE #246: dockerfile build failed
os: ubuntu 20.04

when I run 
```shell
docker build docker/GPU/ -t mmdeploy:GPU
```

failed, what should I do?

```shell

-- Build files have been written to: /root/workspace/mmdeploy/build/install/example/build
[ 10%] Building CXX object CMakeFiles/ocr.dir/ocr.cpp.o
[ 20%] Linking CXX executable ocr
/usr/bin/ld: warning: libcuda.so.1, needed by /root/workspace/mmdeploy/build/install/lib/libmmdeploy_cuda_device.so, not found (try using -rpath or -rpath-link)
/usr/bin/ld: /root/workspace/mmdeploy/build/install/lib/libmmdeploy_cuda_device.so: undefined reference to `cuCtxGetCurrent'
collect2: error: ld returned 1 exit status
make[2]: *** [CMakeFiles/ocr.dir/build.make:172: ocr] Error 1
make[1]: *** [CMakeFiles/Makefile2:91: CMakeFiles/ocr.dir/all] Error 2
make: *** [Makefile:91: all] Error 2
The command '/bin/sh -c cd /root/workspace/mmdeploy &&    rm -rf build/CM* &&    mkdir -p build && cd build &&    cmake ..         -DMMDEPLOY_BUILD_SDK=ON         -DCMAKE_CXX_COMPILER=g++         -Dpplcv_DIR=/root/workspace/ppl.cv/cuda-build/install/lib/cmake/ppl         -DTENSORRT_DIR=${TENSORRT_DIR}         -DMMDEPLOY_BUILD_SDK_PYTHON_API=ON         -DMMDEPLOY_TARGET_DEVICES="cuda;cpu"         -DMMDEPLOY_TARGET_BACKENDS="trt"         -DMMDEPLOY_CODEBASES=all &&    cmake --build . -- -j$(nproc) && cmake --install . &&    cd install/example  && mkdir -p build && cd build &&    cmake -DMMDeploy_DIR=/root/workspace/mmdeploy/build/install/lib/cmake/MMDeploy .. &&    cmake --build . && export SPDLOG_LEVEL=warn &&    if [ -z ${VERSION} ] ; then echo "Built MMDeploy master for GPU devices successfully!" ; else echo "Built MMDeploy version v${VERSION} for GPU devices successfully!" ; fi' returned a non-zero code: 2

```

### PR DESCRIPTION
**Pull request by lvhan028**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily receiving feedbacks. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

Fix #246 

## Modification

docker/GPU/Dockerfile
1. remove cmake installation since the default cmake is 3.14.4 which is enough for building mmdeploy
2. set ppl.cv 0.6.2 as the default version
3. use `git clone` instead of `wget`, otherwise it cannot generate `PPLCV_VERSIONI_*`
4. add the path of stub libcuda.so to LD_LIBRARY_PATH, so that it can find libcuda.so during compilation

### CODE REVIEW

File: docker/GPU/Dockerfile [hunk 1]
```diff
@@ -51,14 +51,6 @@ RUN wget https://github.com/microsoft/onnxruntime/releases/download/v${ONNXRUNTI
...
 RUN cp -r /usr/local/lib/python${PYTHON_VERSION}/dist-packages/tensorrt* /opt/conda/lib/python${PYTHON_VERSION}/site-packages/
 
-### update cmake
-RUN  wget https://github.com/Kitware/CMake/releases/download/v${CMAKE_VERSION}/cmake-${CMAKE_VERSION}.tar.gz &&\
-    tar -zxvf cmake-${CMAKE_VERSION}.tar.gz &&\
-    cd cmake-${CMAKE_VERSION} &&\
-    ./bootstrap &&\
-    make &&\
-    make install
-
```
Comment (AllentDan) [inline docker/GPU/Dockerfile:61]: The `ARG  CMAKE_VERSION` could be removed too.

Comment (lvhan028) [reply docker/GPU/Dockerfile:61]: > The `ARG CMAKE_VERSION` could be removed too.

Done

**Review by irexyc — APPROVED** [review]
build success with proxy.

**Review by AllentDan — APPROVED** [review]
LGTM

**Pull request merged**